aten/src/ATen/native/quantized/cpu/kernels/QuantizedOpKernels.cpp

```diff
@@ -1563,6 +1563,17 @@ void _qavg_pool_nhwc_kernel(
           float multiplier = input_scale / output_scale / divide_factor;
           int input_zero_point_m_size = -input_zero_point * size;
 
+          if (size == 0) {
```

Comment:
Looks like this is causing an ASAN failure and another quantization test failure. What's going on here?

Reply:
The problem was with the AVX2 code when the kernel window is outside the non-padded region. In this case size = 0 so I added a fast-path which also avoids unnecessary loop control logic.

Reply:
Is this necessary? There's no tests in the PR for this quantized code path.

Reply:
removed